bower.json

```diff
@@ -23,5 +23,8 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
+  "dependencies": {
```

Comment:
axios is not the only dependency we have. Take a look at our `package.json` file.